src/DropdownMenu.js

```diff
@@ -16,11 +70,19 @@ const DropdownMenu = React.createClass({
         'dropdown-menu-right': this.props.pullRight
       };
 
+    // Remove outline on focus by default
+    let style = this.props.style || {};
+    style.outline = style.outline || 'none';
+
+    // Adding tabIndex -1 to allow for focus, but not tabable
     return (
         <ul
+          tabIndex="-1"
```

Comment:
Ah, yeah, I was a bit unsure about the style prop.

Hm hm... this is a tricky one. 

*Authors can use either the JavaScript focus() method to move focus to the appropriate element in the widget, or they can use a WAI-ARIA property called aria-activedescendant on the actual widget container to indicate which descendant element in the widget has focus. Use the following procedure when focus is completely dependent on the use of tabindex to provide focus in a widget:

Set tabindex="0" on the current active descendant in the container widget while setting tabindex="-1" on all the other descendant elements of the widget.

As the user navigates (e.g., arrows) away from an item, set tabindex="-1" on the old item, and tabindex="0" on the new.

Use the JavaScript focus() method to put focus on the new item, whose tabindex="0".*

Hm, if I were to build a dropdown from scratch I would indeed use tabIndex='-1' on each `li` element, but bootstrap relies on focusing the `a` elements in the list to style correctly. 

This implementation follows the practice of original bootstrap at least, so yeah... going away from `a` focus to `li` with tabIndex="-1" to focus those will not behave like Boostrap regarding styles.